OfficeApp3Web/Content/AppV2.css

```diff
@@ -483,7 +483,7 @@ li.listItem:hover{
     border:none;
     font-size:15px;
     color:white;    
-    height: auto;
+    height: 25px;

```

Comment:
I guess this is to fix text trimmed issue, right? The bug still exists on Chrome.